src/components/WebhookErrorComponent/WebhookErrorComponent.tsx

```diff
@@ -0,0 +1,24 @@
+import { ErrorResponse } from '@bsv/spv-wallet-js-client';
+import { ErrorComponentProps } from '@tanstack/react-router';
+import { CustomErrorComponent } from '@/components';
+import { Ban } from 'lucide-react';
+
+const WebhooksDisabled = () => (
+  <div className="flex flex-col w-full h-[80vh] justify-center items-center">
+    <p>
+      <Ban className="size-44 text-gray-500" />
+    </p>
+    <p className="mt-4 text-gray-500">Notification feature is not enabled</p>
+  </div>
+);
+
+export const WebhookErrorComponent = ({ error }: ErrorComponentProps) => {
+  if (error instanceof ErrorResponse) {
+    const content = JSON.parse(error.content);
+    if (content.code === 'error-notifications-disabled') {
+      return <WebhooksDisabled />;
+    }
+  }
+
+  return <CustomErrorComponent error={error} />;
+};
```

Comment:
```suggestion
export const WebhookErrorComponent = ({ error }: ErrorComponentProps) => {
  const errorCode = useMemo(() => {
    if (!(error instanceof ErrorResponse)) {
      return null;
    }

    try {
      const content = JSON.parse(error.content);
      return content?.code || null;
    } catch {
      return null;
    }
  }, [error]);

  if (errorCode === 'error-notifications-disabled') {
    return <WebhooksDisabled />;
  }

  return <CustomErrorComponent error={error} />;
};
```

Reply:
Are you sure it will boost performance significantly so it is really the case?

Reply:
Boost performance maybe, but for sure handles exceptions when the content is not a real json.